drivers/storage/portworx/deployment.go

```diff
@@ -287,7 +287,7 @@ func (p *portworx) GetStoragePodSpec(
 		}
 	}
 
-	if pxutil.IsTelemetryEnabled(cluster.Spec) {
+	if pxutil.IsTelemetryEnabled(cluster.Spec) && !pxutil.RunCCMGo(cluster) {
```

Comment:
Can we not use `pxutil.IsCCMGoSupported` here?

Reply:
addressed offline.